lib/pipefy_message/providers/aws_client/sns_broker.rb

```diff
@@ -21,7 +21,7 @@ def initialize(_opts = {})
 
           @topic_arn_prefix = ENV.fetch("AWS_SNS_ARN_PREFIX", "arn:aws:sns:us-east-1:000000000000:")
           @is_staging = ENV["ASYNC_APP_ENV"] == "staging"
-        rescue StandardError
+        rescue StandardError => e
```

Comment:
I had just included this in my own MR when I saw this :joy: Well, I guess that makes it easier to rebase when one is merged before the other :smile: